cypress/integration/homepage_spec.js

```diff
@@ -1,5 +1,209 @@
-describe('Feedback Loop login flows', () => {
-  it('Should confirm that true is equal to true', () => {
-    expect(true).to.equal(true)
+import {
+  titleGift,
+  catPic,
+  dogPic,
+  foxPic,
+  otterPic
+} from '../fixtures/srcData'
+
+describe('Home Page', () => {
+  beforeEach(() => {
+    cy.intercept('https://gifty-backend-rails.herokuapp.com/api/v1/users/1/friends', {
```

Comment:
It looks like this intercept was used for the friendPage_spec too. This could be set as a command in the support folder and be reusable. Future us refactoring exercises :wink: